activerecord/test/cases/adapters/postgresql/legacy_migration_test.rb

```diff
@@ -0,0 +1,49 @@
+require "cases/helper"
+
+class PostgresqlLegacyMigrationTest < ActiveRecord::PostgreSQLTestCase
+  class GenerateTableWithoutBigserial < ActiveRecord::Migration::Compatibility::V5_0
+    def change
+      create_table :legacy_integer_pk do |table|
```

Comment:
Should test whether `Compatibility::V5_0`'s primary key is auto increment (serial) or not.

```ruby
  assert LegacyIntegerPk.create!
```